pkg/collector/corechecks/cluster/ksm/kubernetes_state_aggregators.go

```diff
@@ -156,5 +156,21 @@ var (
 			"statefulset.count",
 			[]string{"namespace"},
 		),
+		"kube_cronjob_labels": newCountObjectsAggregator(
+			"cronjob.count",
+			[]string{"namespace"},
+		),
+		"kube_endpoint_labels": newCountObjectsAggregator(
+			"endpoint.count",
+			[]string{"namespace", "endpoint"},
+		),
+		"kube_horizontalpodautoscaler_labels": newCountObjectsAggregator(
+			"hpa.count",
+			[]string{"namespace", "horizontalpodautoscaler"},
+		),
+		"kube_verticalpodautoscaler_labels": newCountObjectsAggregator(
+			"vpa.count",
+			[]string{"namespace", "verticalpodautoscaler"},
+		),
```

Comment:
The namespace + name aggregation would result in unique context for each object (high cardinality), I think we just need to aggregate by `namespace` like the `statefulsets`, `deployments`, `daemonsets` counters defined above.